app/api/routers/breeze_buddy/telephony/answer/handlers.py

```
@@ -0,0 +1,401 @@
+    call_id, from_number, to_number = _extract_call_params(params, provider)
+    logger.info(f"[{tag}] call_id={call_id}, from={from_number}, to={to_number}")
+
+    if not call_id:
+        logger.error(f"[{tag}] Missing call ID")
+        return _error_response(provider, "Missing call identifier", 400)
+
+    # Plivo-specific: start recording
+    if provider == "plivo":
+        try:
+            start_call_recording(call_id)
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)